src/index.js

```diff
@@ -9,6 +12,26 @@ export function entitize(json) {
   return { entities };
 }
 
+export function findAssociated(store, resourceType, id, assocation) {
+  const chosenAssociation = get(
+    store,
+    ["entities", resourceType, id, "relationships", assocation],
+    {}
+  );
+
+  if (isArray(chosenAssociation)) {
+    return map(chosenAssociation, (association) => {
+      return findById(store, association.type, association.id);
+    });
+  } else {
+    return findById(store, chosenAssociation.type, chosenAssociation.id);
```

Comment:
yeah I thought it was weird to but after some thought I think I'd rather keep it this way. I'm pretty sure that JSON API specs that `has one` or `belongs to` relationships are stored as `objects` while a `has many` returns an `array`. I guess I wanted to stick with that api. I can see that checking for type from here on out would always suck and I'm still unsure what the correct implementation is.